azkaban-web-server/src/web/js/azkaban/util/job-status.js

```diff
@@ -16,7 +16,7 @@
 
 var statusList = ["FAILED", "FAILED_FINISHING", "SUCCEEDED", "RUNNING",
   "WAITING", "KILLED", "DISABLED", "READY", "CANCELLED", "UNKNOWN", "PAUSED",
-  "SKIPPED", "QUEUED", "FAILED_SUCCEEDED", "DISPATCHING"];
+  "SKIPPED", "QUEUED", "FAILED_SUCCEEDED", "DISPATCHING", "POD_FAILED"];
```

Comment:
Can you please validate that Kill link is not coming on Flow execution page and prepare execution link is coming instead? The behavior should be same as any other final state.